src/scss/_font_pairing_casual.scss

```diff
@@ -0,0 +1,34 @@
+html.emc-font-pairing--casual {
+  font-family: $font-face-body-casual;
+}
+
+h1.emc-font-pairing--casual, .emc-h1.emc-font-pairing--casual,
+h2.emc-font-pairing--casual, .emc-h2.emc-font-pairing--casual,
+h3.emc-font-pairing--casual, .emc-h3.emc-font-pairing--casual,
+h4.emc-font-pairing--casual, .emc-h4.emc-font-pairing--casual,
+h5.emc-font-pairing--casual, .emc-h5.emc-font-pairing--casual,
+h6.emc-font-pairing--casual, .emc-h6.emc-font-pairing--casual,
+.emc-page-title.emc-font-pairing--casual,
+.emc-display-heading.emc-font-pairing--casual,
+.emc-block-heading.emc-font-pairing--casual,
+.emc-card__heading.emc-font-pairing--casual,
+.emc-news-card.emc-font-pairing--casual  {
+  font-family: $font-face-heading-casual;
+}
+
+footer.emc-footer.emc-font-pairing--casual h2,
+.emc-card__footer.emc-font-pairing--casual,
+.emc-pager.emc-font-pairing--casual,
+.emc-btn.emc-font-pairing--casual,
+.emc-breadcrumb.emc-font-pairing--casual,
+.emc-modal.emc-font-pairing--casual {
+  font-family: $font-face-body-casual;
+}
+
+emc-event-card__title.emc-font-pairing--casual a {
+  font-family: $font-face-heading-casual;
+}
+
+a.emc-tab-link.emc-font-pairing--casual {
+  font-family: $font-face-body-casual;
+}
```

Comment:
For the "casual" font pairing, I think that the selectors have to be a bit different to avoid the need for the developer to specify the "emc-font-pairing--casual" on each heading.  They should just be able to put it on the "body" tag and have all of the headings change.  To do this, you would want the selectors in this file to be something like this:

.emc-font-pairing--casual h1,
.emc-font-pairing--casual h2,
.emc-font-pairing--casual h3,
...etc...

You will have to test it to make sure, but I think that should work.

Reply:
Makes sense. I wasn't sure how to make this easier. I will try your suggestion and let you know if I run into difficulties.